projects/js-packages/components/components/a8c-svg-title/README.md

```diff
@@ -0,0 +1,16 @@
+# A8C SVG Title
```

Comment:
Could we use a more descriptive name for the component, like `automattic-byline-logo`? I'm not sure having `svg` in the component name is necessary since that's self-evident.

Reply:
renamed in 25d1d9583